Lib/test/test_sax.py

```diff
@@ -774,8 +774,8 @@ class StreamReaderWriterXmlgenTest(XmlgenTest, unittest.TestCase):
     fname = support.TESTFN + '-codecs'
 
     def ioclass(self):
-        writer = codecs.open(self.fname, 'w', encoding='ascii',
-                             errors='xmlcharrefreplace', buffering=0)
+        writer = open(self.fname, 'w', encoding='ascii',
```

Comment:
I don't understand why test_sax tests the StreamReaderWriter class.

It seems like io.TextIOWrapper is not tested, whereas it's more common than codecs.StreamReaderWriter.